code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -351,7 +334,11 @@
 	var/mob/living/carbon/human/H = M
 
 	// Prepare their return
-	prisoner_timer_handle = addtimer(CALLBACK(src, PROC_REF(handle_target_return), M, T), prison_time, TIMER_STOPPABLE)
+	if(isAntag(M))
```

Comment:
I know ERT/Deathsquad *shouldnt* be contracted, but its good to make that clear in the code too.
```suggestion
	if(M.mind.special_role && !(M.mind.special_role in list(SPECIAL_ROLE_ERT, SPECIAL_ROLE_DEATHSQUAD))
```